pkg/cache/clusterqueue.go

```diff
@@ -372,6 +373,10 @@ func (c *ClusterQueue) inactiveReason() (string, string) {
 			reasons = append(reasons, "MultipleSingleInstanceControllerChecks")
 		}
 
+		if c.hasClusterQueueLevelAdmissionCheckAppliedAtFlavorLevel {
+			reasons = append(reasons, "MultiKueueAdmissionCheckAppliedPerFlavor")
```

Comment:
```suggestion
			reasons = append(reasons, "FlavorIndependentAdmissionCheckAppliedPerFlavor")
```
Let's call it that way since the condition checked in the if is more generic